controllers/multicluster/remotecluster/controller.go

```diff
@@ -189,26 +201,36 @@ func (r *Reconciler) ensureHelmRelease(ctx context.Context, obj *v1alpha1.Remote
 	if err != nil {
 		return "", err
 	}
-	release, err := helmClient.GetRelease(obj.Spec.Namespace, chaosMeshReleaseName)
+	_, releaseErr := helmClient.GetRelease(obj.Spec.Namespace, chaosMeshReleaseName)
+	if releaseErr != nil && !errors.Is(releaseErr, driver.ErrReleaseNotFound) {
+		return "", releaseErr
+	}
+	chart, err := helm.FetchChaosMeshChart(ctx, obj.Spec.Version, config.ControllerCfg.LocalHelmChartPath)
 	if err != nil {
-		if errors.Is(err, driver.ErrReleaseNotFound) {
-			chart, err := helm.FetchChaosMeshChart(ctx, obj.Spec.Version, config.ControllerCfg.LocalHelmChartPath)
-			if err != nil {
-				return "", err
-			}
+		return "", err
+	}
 
-			values := make(map[string]interface{})
-			if obj.Spec.ConfigOverride != nil {
-				err = json.Unmarshal(obj.Spec.ConfigOverride, &values)
-				if err != nil {
-					return "", err
-				}
-			}
-			release, err = helmClient.UpgradeOrInstall(obj.Spec.Namespace, chaosMeshReleaseName, chart, values)
-			if err != nil {
-				return "", err
-			}
-		} else {
+	values := make(map[string]interface{})
+	if obj.Spec.ConfigOverride != nil {
+		configStr := strings.Trim(string(obj.Spec.ConfigOverride), "\"")
+		configByte, err := base64.StdEncoding.DecodeString(configStr)
```

Comment:
The type of `configOverride` is `json.RawMessage`, not `base64`. I think you're using the one that suits your situation here, but it causes incompatibility. You should revert this change.